CONTRIBUTING.md

```diff
@@ -40,3 +40,15 @@ Creating very large pull requests may significantly increase review time or even
 
 At this time, we will not be accepting contributions that only fix spelling or grammar errors in documentation, code or
 elsewhere.
+
+### `sncast` Guidelines
+
+#### Command outputs
+
+Please follow these rules when creating outputs for `sncast`:
+
+- Use an imperative tone
+- Keep your message concise and to the point
+- When displaying config, use `key: value` format
+- If the executed command has a natural successor-command, display it as hint in the output
```

Comment:
Can we add an example of a good follow up command?

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/2843/commits/7b478a6429dae346040478427b58d950b0cfc8c9